index.html

```diff
@@ -5,217 +5,219 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-<link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
-    <title>portofolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/69bb18a36d.js" crossorigin="anonymous"></script>
+    <title>Document</title>
 </head>
 <body>
-    
-       <div class="container">
-        <section>
-           <div class="card">
-           <header>
-            <div class="fcontainer">
-                <a class="logo">Saba</a>
-                <nav id="items">
-                  <i class="fa fa-bars"></i>
-                </nav>
-              </div>
-             </header>
-           <div class="fcontent">
-            <h1>I'M Saba Ahmad</h1>
-            <h2>Glad to see</h2>
-            <h3>you!</h3>   
+    <header class="nav-bar">
+           <a class="logo" href="#">Saba Ahmad</a>
+            <nav>
+                <ul class="nav-items">
+                    <li>portofolio</li>
+                    <li>about</li>
+                    <li>contact</li>
+                </ul>
+            </nav>
+    </header>
+    <section>
+        <div class="introduction">
+        <h1 class="title">I'm Saba Ahmad <br>
+         Glad to see you!</h1>
+       <p class="paragraph">I’m a fullstack web developer! I can help you build a website, feature or website Look through <br> some of my work and experience! If you like what you see and have a project you need <br> coded, don’t hestiate to contact me.</p>
+       <h2 class="connect">Let's Connect</h2>
```

Comment:
- [ ] Kindly correct "LET'S CONNECT" instead of "Let´s Connect" to look similar to the Figma template.